### ISSUE TITLE
Firestore

### ISSUE THREAD

**Issue by khalednaz** [Syed-Waleed-Shah/infinite_scroll#4]
Is it possible to use this package with firebase to retrieve the data and if so is there an example?

**Comment by Syed-Waleed-Shah** [comment]
Yes it is possible. Fetch your firestore data in chunks and parse the data
and display it in the list as shown in the example.
